advanced/am-pattern-4/templates/mi/instance-1/wso2am-pattern-4-mi-deployment.yaml

```diff
@@ -0,0 +1,109 @@
+# Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Do we need a separate directory for for MI(`instance-1`) as we only have a single MI deployment?